cliquet_docs/reference/notifications.rst

```diff
@@ -3,30 +3,64 @@
 Notifications
 #############
 
-Knowing when a bucket or a collection has been modified is very
+Knowing collection has been modified is very
```

Comment:
> Also, this event is triggered if we change a bucket, no ?

There is no notion of bucket in *Cliquet*.

So I propose:

Knowing some records have been modified in a resource is very useful..